### ISSUE TITLE
No Such Method Error

### ISSUE THREAD

**Issue by Joyliu290** [Shopify/mobile-buy-sdk-android#437]
Hi, I've seen someone asking this error few days ago, but the answer wasn't clear so sorry if I'm asking this again.

I have my own app that integrates Shopify's API. To test things, below shows a sample code posted by you guys on how to get the shop name:

![shopify_error](https://user-images.githubusercontent.com/23625527/29686366-ae47dbb2-88e6-11e7-8537-80c41c83521c.png)

I'm getting this error:

![actual_error](https://user-images.githubusercontent.com/23625527/29686475-10261588-88e7-11e7-92ca-612ef521c9ec.png)

Thank You!

**Comment by sav007** [comment]
Are you using retrolambda in your project?

**Comment by sav007** [comment]
@Joyliu290 could you please provide more info about your case, just to understand what was the issue. I only assume that you didn't have retrolambda in your project before?

**Comment by Joyliu290** [comment]
@sav007 the issue was that I followed Android Studio's instruction on integrating Java 8 by migrating from Jack (didn't know retrolambda existed). However I must've messed up in the process or something happened but this issue happened and to debug I created another project without any integration of Java 8 and the code that I showed to fetch shop's name worked fine. I believe not having retrolambda and messing up the Jack is my issue.
